--- kcometen3-1.1/admin/cvs.sh~	2007-12-25 20:37:54.000000000 +0200
+++ kcometen3-1.1/admin/cvs.sh	2007-12-25 20:38:52.181943226 +0200
@@ -64,7 +64,7 @@
     echo "*** KDE requires automake 1.6.1 or newer"
     exit 1
     ;;
-  automake*1.6.* | automake*1.7* | automake*1.8.* | automake*1.9* ) : ;;
+  automake*1.6.* | automake*1.7* | automake*1.8.* | automake*1.9* | automake*1.10* ) : ;;
   "" )
     echo "*** AUTOMAKE NOT FOUND!."
     echo "*** KDE requires automake 1.6.1 or newer"
