diff -urN a/src/nntpthreadsocket.cpp b/src/nntpthreadsocket.cpp
--- a/src/nntpthreadsocket.cpp	2004-11-17 00:04:18.000000000 +0100
+++ b/src/nntpthreadsocket.cpp	2005-01-06 00:15:03.598562512 +0100
@@ -1983,14 +1983,14 @@
 	if (!m_sendCmd(cmd, NntpThreadSocket::list))
 		return false;	
 	
-	while (!(cancel) && line[0] != '.') {
+	while (!(line[0] == '.') && !(line[1] == '\0')  ) {
 		if (!waitLine())
 			return false;
 		
 		
 		
 		while (m_readLine()) {
-			if (line[0] == '.')
+			if ( (line[0] == '.') && (line[1] == '\0') )
 				break;
 			else {
 								
